rpc_jobs/standard_job_premerge_release.yml

```diff
@@ -0,0 +1,85 @@
+- project:
+    name: "RE-Release-PR"
+    repo:
+      - rpc-openstack:
+          URL: "https://github.com/rcbops/rpc-openstack"
+          BRANCH: "master"
+      - rpc-openstack:
+          URL: "https://github.com/rcbops/rpc-openstack"
+          BRANCH: "newton"
+    jobs:
+      - 'RE-Release-PR_{repo}-{BRANCH}'
+
+
+- job-template:
+    RPC_GATING_BRANCH: master
+    name: 'RE-Release-PR_{repo}-{BRANCH}'
```

Comment:
Should we call this `PR_RE-Release-{repo}-{BRANCH}` to keep it a bit more standard with other job names?

Reply:
I wanted to, but then the job name linter failed and I didn't want to go down the road of trying to figure out the complex regex.